src/lib.rs

```diff
@@ -152,6 +153,9 @@ pub extern "C" fn svsm_main() -> ! {
     // Load BIOS
     start_bios();
 
+    // Initialize resources for SNP_GUEST_REQUEST messages
+    snp_guest_request_init();
```

Comment:
This should be called before start_bios() so that we have all the initialization out of the way before starting vmpl1.